resolver/resolver.cc

```diff
@@ -2050,11 +2039,6 @@ class ResolveSanityCheckWalk {
     unique_ptr<ast::Expression> postTransformMethodDef(core::MutableContext ctx, unique_ptr<ast::MethodDef> original) {
         ENFORCE(original->symbol != core::Symbols::todo(), "These should have all been resolved: {}",
                 original->toString(ctx));
-        for (auto &arg : original->args) {
-            if (auto optionalArg = ast::cast_tree<ast::OptionalArg>(arg.get())) {
-                ENFORCE(!optionalArg->default_);
```

Comment:
can we check that it's now always an `EmptyTree`?

Reply:
no, in some cases it is left (like in the included test case) and in overloaded methods